kong/plugins/basic-auth/api.lua

```diff
@@ -52,5 +52,29 @@ return {
     DELETE = function(self, dao_factory)
       crud.delete(self.basicauth_credential, dao_factory.basicauth_credentials)
     end
+  },
+  ["/basic-auth/"] = {
+    GET = function(self, dao_factory)
+      crud.paginated_set(self, dao_factory.basicauth_credentials)
+    end
+  },
+  ["/basic-auth/:credential_id/consumer"] = {
+    before = function(self, dao_factory, helpers)
+      local credentials, err = dao_factory.basicauth_credentials:find_all { id = self.params.credential_id }
```

Comment:
style: we should avoid exceeding the 80 columns limit for long lines as documented in our contribution guidelines. Thanks!